cc_shim_test.go

```diff
@@ -0,0 +1,114 @@
+//
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package virtcontainers
+
+import (
+	"testing"
+)
+
+var testShimPath = "/tmp/bin/cc-shim-mock"
+var testProxyURL = "foo:///foo/clear-containers/proxy.sock"
+
+func testCCShimStart(t *testing.T, pod Pod, params ShimParams, expectFail bool) {
+	s := &ccShim{}
+
+	pid, err := s.start(pod, params)
+	if expectFail {
+		if err == nil || pid != -1 {
+			t.Fatal("This test should fail")
```

Comment:
Since `testCCShimStart()` is called by other functions and since the testing framework doesn't display a stack trace on `t.Fatal()`, shouldn't these `t.Fatal()` calls atleast display the values of `pod`, `params` and `expectFail`?